repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc

```diff
@@ -0,0 +1,133 @@
+= OpenTracing Service
+
+The OpenTracing service (http://opentracing.io/) is a service in *Repose* which allows for sending tracing data to a Tracing service.
+
+The initial release of the OpenTracing service only supports using the Jaeger (http://jaeger.readthedocs.io/en/latest/) Tracer.
+However, other Tracers can be integrated in future OpenTracing service updates.
+By default, the service is disabled, but it can be enabled by including an `open-tracing.cfg.xml` configuration.
+The origin service can simply pass the Tracer-specific header to downstream services and requests will be traced across application boundaries.
+
+== General Service information
+* *Default Configuration:* open-tracing.cfg.xml
+* *Released:* vTBD
+* link:../schemas/open-tracing.xsd[Schema]
+
+== Tracer implementation detail
+
+This section will cover all Tracers that are supported by the OpenTracing service.
+We will be adding to this list as more tracers as supported.
+
+Current support:
+
+.Table Tracer information
+|===
+|Tracer Name |Tracer Header    |Extraction examples
+
+|*Jaeger*
+|`uber-trace-id`
+|https://github.com/jaegertracing/jaeger-client-java/tree/master/jaeger-apachehttpclient
+|===
+
+== Examples
+=== Basic Configuration
+This configuration will send all traces to the default HTTP collector at `http://localhost:14268`.
```

Comment:
Just so we don't forget, the default endpoint has been dropped.

Reply:
Updated.